src/index.html

```diff
@@ -7,8 +7,41 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <nav class="navigation">
+        <a href="index.html" class="icon">
+          <img src="images/logo.png"
+          class="nav"
+          alt="Mate Academy">
+        </a>
+        <ul class="nav__list">
+          <a href="#section1"
+             class="margin__nav is-active">Apple</a>
+          <a href="#section2" class="margin__nav ">Samsung</a>
+          <a href="#section3" class="margin__nav ">Smartphones</a>
+          <a href="#section4"
+             class="margin__nav" data-qa="hover">Laptops & Computers</a>
+          <a href="#section5" class="margin__nav ">Gadgets</a>
+          <a href="#section6" class="margin__nav ">Tablets</a>
+          <a href="#section7" class="margin__nav ">Photo</a>
+          <a href="#section8"class="margin__nav right_margin ">Video</a>
+        </ul>
+      </nav>
+    </header>
+    <section id="section1">Apple</section>
```

Comment:
Rename id`s to more meaningful names, don't use naming like name1, name2, name3, etc. Its bad practice and it makes your code dirty